src/deepsparse/transformers/pipelines/text_generation.py

```diff
@@ -829,6 +829,11 @@ def engine_forward(
                     generated_tokens.append(token)
                     generated_logits.append(logits)
 
+                    if session.total_num_processed_tokens > session.capacity:
+                        # if the kv cache is full, stop generation
+                        finished_reason.append(FinishReason.CAPACITY)
+                        break
```

Comment:
I patched this PR and there's an off-by-one error somewhere. A 65th token is being inserted into the KV cache.
Should this be 
```if session.total_num_processed_tokens >= session.capacity:```?

Reply:
@SageMoore could you double check this on our end as well, any possibility that we're walking the cache an extra time or something similar?

Reply:
No, you are right @tlrmchlsmth . It needs to be `>=`, just checked